Library/Formula/qemu.rb

```diff
@@ -10,6 +10,12 @@ class Qemu < Formula
     sha256 "805da9cd1c6193e345245a4c33dc090fd8b541190c56d64acb927cb37e794d50" => :mountain_lion
   end
 
+  patch do
+    # Fix VENOM
```

Comment:
Can you add more detail in this comment e.g. where this patch is from/applied, when it will be needed until and what VENOM is? Thanks!